packages/grafana-ui/src/components/RenderUserContentAsHTML/RenderUserContentAsHTML.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { HTMLAttributes, PropsWithChildren } from 'react';
+
+export interface RenderUserContentAsHTMLProps extends Omit<HTMLAttributes<HTMLElement>, 'dangerouslySetInnerHTML'> {
+  component?: React.ElementType;
+  content: string;
+}
```

Comment:
Tweaked the types here slightly to allow for `<RenderUserContentAsHTML component={Button} content="sample content" />`

Reply:
Thanks @joshhunt. Can I get a ✅ :)